system-contracts/contracts/precompiles/test-contracts/Keccak256Mock.yul

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
How does it differ from the original one?

Reply:
It should just have a different bytecode hash to simulate the upgrade of keccak